src/test/java/racing/CarDrivingTypesTest.java

```diff
@@ -0,0 +1,104 @@
+package racing;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class CarDrivingTypesTest {
+    @Test
+    @DisplayName("이동상태를 추가한다")
+    void addTest() {
+        List<CarDrivingType> carDrivingTypeArrayList = new ArrayList<>();
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+
+        CarDrivingTypes carDrivingTypes = new CarDrivingTypes(carDrivingTypeArrayList);
+        assertThat(carDrivingTypes.getGoMovingCount()).isEqualTo(3);
+    }
+
+    @Test
+    @DisplayName("이동상태 GO 카운팅")
+    void CountingOnlyCarDrivingTypeIsGoTest() {
+        List<CarDrivingType> carDrivingTypeArrayList = new ArrayList<>();
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+        carDrivingTypeArrayList.add(CarDrivingType.STOP);
+        carDrivingTypeArrayList.add(CarDrivingType.GO);
+
+        CarDrivingTypes carDrivingTypes = new CarDrivingTypes(carDrivingTypeArrayList);
+        assertThat(carDrivingTypes.getGoMovingCount()).isEqualTo(4);
+    }
+
+    @Test
+    @DisplayName("이동상태 STOP 카운팅")
+    void CountingOnlyCarDrivingTypeIsStopTest() {
```

Comment:
```suggestion
    @Test
    @DisplayName("STOP 을 1개만 추가했다면, 이동상태 STOP 이 1개로 카운팅 되어야 한다.")
    void CountingOnlyCarDrivingTypeIsStopTest() {
```

DisplayName 을 작성할 때는, expected value 를 유추할 수 있게끔 작성해주시면 좋습니다.